src/gpg/contact/delete.py

```diff
@@ -0,0 +1,33 @@
+import gpg
+import sys
+import os
+from fn.print_key import print_key
+
+
+def delete(contacts, force):
+    try:
+        c = gpg.Context()
+        for contact in contacts:
+            keys = list(c.keylist(contact))
+            ans = "n"
+            for key in keys:
+                if(not force):
```

Comment:
Why still use brackets for if `expr`? Please keep it consistent.

Reply:
cool I have opened #79 also for this. This gets missed as I am using flake8 which doesn't detect this by default.